extras/docker/fromsource/heketi-start.sh

```diff
@@ -5,58 +5,58 @@
 
 echo "Setting up heketi database"
 
-if [[ -d ${HEKETI_PATH} ]]; then
+if [[ -d "${HEKETI_PATH}" ]]; then
```

Comment:
oh, this is already in the patchset.
these fixes need to be squashed with the original introduction earlier in this set.